frontend/src/Header/Header.css

```diff
@@ -0,0 +1,57 @@
+.flexcontainer {
```

Comment:
Overall question, if we're doing the scss stuff does it make sense to check in the css files?

Maybe we should ignore them to make sure people use the generated files?

Reply:
yeah, I think that's a good call. Crossed my mind too.